src/services/room-service.js

```diff
@@ -91,6 +93,10 @@ export default class RoomService {
 
     const owner = await this.userService.getUserById(room.owner);
 
+    if (owner.email.toLowerCase() === lowerCasedEmail) {
+      throw new BadRequest('The owner may not invite herself as a member');
```

Comment:
I guess if the owner is male, they also cannon invite themselves 😅 

How about
`throw new BadRequest('Invited room member is the same as room owner');`
?

Reply:
Fixed to the new message.